.gitignore

```diff
@@ -0,0 +1,11 @@
+## IDE Specific
+.idea/
+out/s
+*.iml
+*.ipr
+*.iws
+
+.settings/
+.classpath
+.project
+s
```

Comment:
This project will not be edited in IDE, so I think this configuration is unnecessary.